.chloggen/strict-unmarshal.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

```suggestion
change_type: 'breaking'
```
Technically this is a breaking change since users with invalid keys will now see errors on startup.

---

Makes sense. Committing change. Thanks